library/linux/src/embedder.cc

```diff
@@ -275,6 +277,10 @@ GLFWwindow *CreateFlutterWindow(size_t initial_width, size_t initial_height,
   FlutterEmbedderState *state = new FlutterEmbedderState();
   state->plugin_handler = std::make_unique<PluginHandler>(engine);
   state->engine = engine;
+
+  state->key_event_handler = std::make_unique<KeyEventHandler>();
+  state->key_event_handler->SetBinaryMessenger(state->plugin_handler.get());
```

Comment:
Since you never need to change this, and the handler doesn't really do anything without it, strongly consider making this a constructor argument.